cartridges/int_adyen_SFRA/cartridge/scripts/hooks/payment/processor/adyen.js

```diff
@@ -56,7 +56,14 @@ function Authorize(orderNumber, paymentInstrument, paymentProcessor) {
         CurrentRequest: request,
         PaymentInstrument: paymentInstrument,
         PaymentType: session.custom.paymentType,
-        Issuer: session.custom.issuer
+        Issuer: session.custom.issuer,
+        dob : session.forms.adyPaydata.dob.value,
+        gender : session.forms.adyPaydata.gender.value,
+        houseNumber : session.forms.adyPaydata.houseNumber.value,
+        houseExtension : session.forms.adyPaydata.houseExtension.value,
+        personalNumber : session.forms.adyPaydata.personalNumber.value,
+        ratePayFingerprint : session.custom.ratePayFingerprint,
+        adyenFingerprint : session.forms.adyPaydata.adyenFingerprint.value
```

Comment:
if data does not exists will it be undefined ? should we check if it exists here ?

Reply:
If it does not exist, the value "null" will be assigned.